src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -138,6 +138,8 @@ def ensure_bicep_installation(release_tag=None, target_platform=None, stdout=Tru
 
         os.chmod(installation_path, os.stat(installation_path).st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
 
+        cli_ctx.config.set_value("bicep", "use_binary_from_path", "false")
```

Comment:
Could we add a log/warning to notify the user about the config change if `use_binary_from_path` is not set to `false`?

Reply:
Done 👍